docs/user/trusted-publishers/using-a-publisher.md

```diff
@@ -379,6 +379,11 @@ below describe the setup process for each supported trusted publisher.
 
 === "GitLab CI/CD"
 
+    !!! note
+
+        Currently, only projects hosted on https://gitlab.com are supported. Self-managed
+        instances are not supported.
+
```

Comment:
No strong opinion, but IMO we could drop this one to reduce duplication: the note is already showing on both of the creation flows, so we don't need it on the use flow 🙂

Reply:
done, removed